lib/normalize.js

```diff
@@ -32,6 +33,16 @@ var _splitLocator = function (nemo, locatorString) {
 module.exports = function normalize(nemo, _locator) {
   var locator = _locator;
   var normalizedLocator;
+
+  if (locator.using && locator.value && nemo.wd.By[locator.using]) {
+    normalizedLocator = nemo.wd.By[locator.using](locator.value);
+    if(normalizedLocator.constructor === locator.constructor){
```

Comment:
```
if (locator.using && locator.value && nemo.wd.By[locator.using]) {
    return locator;
  }
```
The above would return on an object literal that was not created with `By` function.  I just discovered a bug with the `nemo.wd.By[locator.using]` line, so I've updated the code and tests by extracting the Locator constructor from a sample By object instead.  Please take a look.